src/components/Note/EditableNoteMentions.tsx

```diff
@@ -135,6 +141,20 @@ const EditableNoteMentions: FC<EditableNoteMentionsProps> = ({
     </style>
   );
 
+  const editNoteMentionsAlert = () => {
+    const mentionStrings = mentionableUsers.map((user) => `@${user.value}`);
+    const userIsMentioned = mentionStrings.some((mention) => initialNoteText.includes(mention));
```

Comment:
Nit: In the `NoteMentions`, we do something similiar. Should we try and match the pattern there?
https://github.com/appfolio/react-gears/blob/master/src/components/Note/NoteMentions.tsx#L112C12-L113C26

e.g. 
```
return initialNoteText.split(/(\s+)/).filter((word) => mentionStrings.includes(word)).length > 0
```